css/templates/e-commerce/page.css

```diff
@@ -328,6 +332,14 @@ amp-selector [option][selected] {
   text-indent: -var(--space-2);
 }
 
+.ampstart-headerbar-fixed-link-wrapper {
+  width: 25px;
```

Comment:
This element is outside of the fixed-link which is an img or text. 

Since we are inlining the SVG into the page, I didn't think using amp-img would be useful?